src/core/main.js

```diff
@@ -358,6 +359,7 @@ class p5 {
       }
 
       this._lastFrameTime = window.performance.now();
+      this._deltaFrame = window.performance.now();
```

Comment:
How do you feel about renaming `_lastFrameTime` and `_deltaFrame` to `_lastTargetFrameTime` and `_lastRealFrameTime`, respectively? It might make it clearer that they are both timestamps instead of differences between timestamps, and also that the former is the time we expected the frame to be drawn at, while the latter is the time it was actually called at.